test/checks/mobile/css-orientation-lock.js

```diff
@@ -3,7 +3,7 @@ describe('css-orientation-lock tests', function() {
 
 	var checkContext = axe.testUtils.MockCheckContext();
 	var origCheck = checks['css-orientation-lock'];
-	var dynamicDoc = document.implementation.createHTMLDocument();
+	var dynamicDoc = document.implementation.createHTMLDocument('ie is dumb');
```

Comment:
Let's use something else for the page title. How about something that describes the page like "CSS Orientation Lock Tests"?